lib/helpers.js

```diff
@@ -45,9 +46,36 @@ function getConfigPath() {
     return CONFIG_FILE_PATH;
 }
 
+function generateDataJson() {
+    const config = getConfig();
+    const data = {
+        timestamp: new Date(),
```

Comment:
[Here's a thread on mocking out `new Date()` in node](https://stackoverflow.com/questions/28504545/how-to-mock-a-constructor-like-new-date?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa). This is kind of what I was suggesting. Another option would be to use `Date.parse` here, perhaps, something like `Date.parse('04 Dec 1995 00:12:00 GMT');`.